scripts/src/public/services/orders.ts

```diff
@@ -231,7 +240,7 @@ function openOrderDbToApi(order: db.Order): OpenOrder {
 
 function orderDbToApi(order: db.Order): Order {
 	if (order.status === "opened") {
-		throw new Error("opened orders should not be returned");
+		throw OpenedOrdersUnreturnable();
```

Comment:
same here: Maybe this should be some assert? it's an internal error and a programming bug